### PR TITLE
fixed facebook-github-bot command duplication

### PR DESCRIPTION
**Pull request by bestander**
clowns

### CODE REVIEW

**Comment by bestander** [comment]
@facebook-github-bot shipit

File: bots/IssueCommands.txt [hunk 1]
```diff
@@ -35,7 +35,3 @@ reopen
 @facebook-github-bot feature
 comment Hey {issue_author}! Thanks for opening the issue, however it looks like a feature request. As noted in the [Issue template](https://github.com/facebook/react-native/blob/master/ISSUE_TEMPLATE.md) we'd like to use the GitHub issues to track bugs only. Can you implement the feature as a standalone npm module? If not consider sending a pull request or a create an entry on [Product Pains](https://productpains.com/product/react-native). It has a voting system and if the feature gets upvoted enough it might get implemented.
 feature
```
Comment (mkonicek) [inline bots/IssueCommands.txt]: This should be "close".

Comment (mkonicek) [inline bots/IssueCommands.txt]: Consider adding "Will close this issue." as a last sentence.

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by mkonicek** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**